R/FeatSelResult.R

```diff
@@ -20,37 +20,33 @@
 #' @rdname FeatSelResult
 NULL
 
-
-makeFeatSelResult = function(learner, control, resampling, x, y, threshold, opt.path) {
-  makeOptResult(learner, control, resampling, x, y, threshold, opt.path, "FeatSelResult")
-}
-
-
 #' @export
 print.FeatSelResult = function(x, ...) {
   catf("FeatSel result:")
 
-  n.feats = length(x$x)
-  printed.features = 10L
-  if (n.feats > printed.features)
-    x = c(head(x, printed.features), "...")
-  else
-    x = head(x, printed.features)
+  shortenX = function(x) {
+    clipString(collapse(x, ", "), 50L)
+  }
 
-  catf("Features (%i): %s", n.feats, collapse(x$x, ", "))
+  if (!all.equal(x$x.bit.names, x$x)) {
+    catf("Bits (%i): %s", length(x$x.bit.names), shortenX(x$x.bit.names))
```

Comment:
Shouldn't that be `messagef`?

Reply:
no thats a printer?

Reply:
catf because a massage is not the same as cat.